Content.Server/Containers/ItemSlot/ItemSlotsComponent.cs

```diff
@@ -0,0 +1,39 @@
+using Content.Shared.Sound;
+using Content.Shared.Whitelist;
+using Robust.Shared.Containers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.ViewVariables;
+using System;
+using System.Collections.Generic;
+
+namespace Content.Server.Containers.ItemSlots
```

Comment:
This really isnt important, but AFAICT theres no reason currently for this to be in server rather than shared.

Whenever interactions & verbs are refactored to support prediction, this would probably need to be moved into shared, so it might as well be moved there already?

This is also true for the system, but its a bit more of a hassle there because you have to use SharedHandsComponent instead of HandsComponent. But all of the HandsComponent functions are just wrappers for functions in SharedHandsComponent anyways, so I think it should still be easy enough to do.